src/views/Onboarding/OnboardingSetup.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+<div>
+  <div class="backup-wallet login-wrapper no-outer-pad" v-if="currentStep === 'backup'">
+    <div class="backup-wallet_top">
+      <CompletedIcon class="backup-wallet_icon" />
+      <h2 class="p-1">Backup your wallet</h2>
+      <p class="m-0">The seed phrase is the only way to restore your wallet. Write it down, verify it and then store it securely.</p>
+    </div>
+    <div class="backup-wallet_bottom">
+      <div class="backup-wallet_seed pt-1">
+        <span v-for="word in seedList" :key="word">{{ word }}</span>
+      </div>
+      <button class="btn btn-primary btn-lg btn-block btn-icon" @click="pushToConfirm">Next</button>
+    </div>
+  </div>
+  <ConfirmSeed v-if="currentStep === 'confirm'" @on-confirm="confirmMnemonic" @on-cancel="currentStep = 'backup'" :mnemonic="mnemonic" />
+  <Congratulations v-if="currentStep === 'congrats'" />
+</div>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+import CompletedIcon from '@/assets/icons/completed.svg'
+import { generateMnemonic } from 'bip39'
+import ConfirmSeed from './SeedPhrase/ConfirmSeed'
+import Congratulations from './SeedPhrase/Congratulations.vue'
+
+export default {
+  data () {
+    return {
+      mnemonic: null,
+      currentStep: 'backup'
+    }
+  },
+  components: {
+    CompletedIcon,
+    ConfirmSeed,
+    Congratulations
+  },
+  created () {
+    this.mnemonic = generateMnemonic()
+  },
+  computed: {
+    ...mapState(['tempPassword']),
+    seedList: function () {
+      return this.mnemonic.split(' ')
+    }
+  },
+  methods: {
+    ...mapActions(['setupWallet', 'createWallet', 'unlockWallet']),
+    async confirmMnemonic () {
+      this.currentStep = 'congrats'
+      const password = this.tempPassword
+      await this.setupWallet({ key: password })
+      await this.createWallet({ key: password, mnemonic: this.mnemonic }) // mnemonic prop can be null to generate new seed
+      setTimeout(() => {
+        this.unlockWallet({ key: password })
+      }, 3500)
+    },
+    pushToConfirm () {
+      this.currentStep = 'confirm'
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+.backup-wallet {
+  padding: 0 !important;
+  overflow: hidden;
+
+  .backup-wallet_bottom {
+    flex: 1;
+    background: #FFFFFF;
+    color: $color-text-primary;
```

Comment:
isn't this default?

Reply:
I guess not lol, text is defaulting to white there.